pages/1.11/deploying-services/index.md

```diff
@@ -13,7 +13,7 @@ enterprise: false
 
 DC/OS uses Marathon to manage processes and services. Marathon is the "init system" for DC/OS. Marathon starts and monitors your applications and services, automatically healing failures.
 
-A native Marathon instance is installed as a part of DC/OS installation. After DC/OS has started, you can manage the native Marathon instance through the **Services** tab of the DC/OS web interface or from the DC/OS CLI with the `dcos marathon` command.
```

Comment:
If you go to your Services tab under 1.11, it says "no running services" - It's not manageable from there.